Cargo.toml

```diff
@@ -97,7 +97,7 @@ wav = ["bevy_internal/wav"]
 # Enable watching file system for asset hot reload
 filesystem_watcher = ["bevy_internal/filesystem_watcher"]
 
-serialize = ["bevy_internal/serialize"]
+serde = ["bevy_internal/serde"]
```

Comment:
why renaming that feature?

Reply:
I renamed it to be more consistent with what @afonsolage proposed above. As we are activating/deactivating both serialization *and* deserialization, we thought that ``serde`` would be a more appropriate name.

What do you think about this?

Reply:
I think I'm weakly in favor of this change, but it needs to be called out in the migration guide as a fairly serious breaking change if we do so.

Reply:
There is a world where Reflect serialization is eventually fully serde independent. And at that point distinguishing between serde and serialization would be a reasonable thing to do. Directly calling out serde does also feel "better" to me generally.

That being said, thats probably a long way off, and I see no reason to rock the boat right now. I think its better to avoid breaking people at this point.

Reply:
Okay, then I will revert this change for now! :)